src/NServiceBus.Core/EndpointCreator.cs

```diff
@@ -97,9 +97,9 @@ void Initialize()
 
             var featureConfigurationContext = new FeatureConfigurationContext(settings, containerComponent.ContainerConfiguration, pipelineComponent.PipelineSettings, routingComponent, receiveConfiguration);
 
-            //note: This is where the settings gets locked since the feature component (unfortunately) uses the settings to store feature state.
-            // Locking happens just before the Features gets "setup"
```

Comment:
@andreasohlund was this comment even correct? because the setup happened before locking the settings I think?

Reply:
Good catch, that comment was wrong